lib/redis/distributed.rb

```diff
@@ -626,6 +626,11 @@ def zscore(key, member)
       node_for(key).zscore(key, member)
     end
 
+    # Get the scores associated with the given members in a sorted set.
+    def zmscore(key, members)
```

Comment:
Why would the distributed version not take `*members` as well? Sounds like a mistake.

Reply:
changed it to variadic parameter